### PR TITLE
Document responses update_bulk_properties endpoint

### PR DESCRIPTION
**Pull request by pmonfort**
This PR document [responses update_bulk_properties endpoint](https://github.com/Wootric/wootric-ai-app-backend/pull/507)

### CODE REVIEW

File: source/includes/cxi/_responses.md [hunk 1]
```diff
@@ -124,3 +124,40 @@ Parameter | Type | Default | Description
...
+
+### HTTP Request
+
+`PUT https://cxi-api.wootric.com/v1/reports/update_bulk_properties`
+
+### HTTP Headers
+
+`Content-Type: multipart/form-data`
+
+### HTTP POST Body
```
Comment (diegoserranoa) [inline source/includes/cxi/_responses.md]: It should be `PUT`, right?

**Review by diegoserranoa — APPROVED** [review]
I just left one question, other than that :shipit: :shipit:

File: source/includes/cxi/_responses.md (hunk 1 above)
Comment (pmonfort) [reply source/includes/cxi/_responses.md]: Ups, great catch, thanks!

**Review by diegoserranoa — APPROVED** [review]
:shipit:

**Pull request merged**